float32 pos_x
float32 pos_y
float32 speed
float32 category
float32 counter

# 0 - pedestrian
# 1 - car
# 2 - unknown
# 3 - tracking
